docs/guides/libraries-usage.md

```diff
@@ -0,0 +1,195 @@
+# Libraries usage
```

Comment:
I'd call it "Lazy Loading Libraries" and just focus on lazy loading

Reply:
I agree. The changes were made.